templates/shortcode-receipt.php

```diff
@@ -148,7 +148,7 @@
 						?>
 
 						<div class="edd_purchase_receipt_product_name">
-							<?php echo esc_html( $item['name'] ); ?>
+							<?php echo esc_html( edd_get_cart_item_name( $item ) ); ?>
 							<?php if( ! is_null( $price_id ) ) : ?>
 							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo edd_get_price_option_name( $item['id'], $price_id, $payment->ID ); ?></span>
```

Comment:
We need to not change this part since the the price ID is added below with it's own HTML.